definitions/variable/environment/biodiversity.yaml

```diff
@@ -0,0 +1,64 @@
+- Terrestrial Biodiversity|Mean Species Abundance:
+    description: Terrestrial biodiversity measured in Mean Species Abundance (MSA)
+    unit: '%'
+    sdg: 15
+    skip-region-aggregation: true
+    weight: Land Cover
+    tier: 3
+- Terrestrial Biodiversity|Mean Species Abundance|Vertebrates:
+    description: Terrestrial biodiversity of vertebrate species measured in Mean Species
+      Abundance (MSA)
+    unit: '%'
+    weight: Land Cover
+    navigate: Terrestrial Biodiversity|MSA|Vertebrates
+    tier: 3
+- Terrestrial Biodiversity|Mean Species Abundance|Plants:
+    description: Terrestrial biodiversity of plant species measured in Mean Species
+      Abundance (MSA)
+    unit: '%'
+    weight: Land Cover
+    navigate: Terrestrial Biodiversity|MSA|Vertebrates
+    tier: 3
+- Terrestrial Biodiversity|Biodiversity Intactness Index:
+    description: Terrestrial biodiversity measured with Biodiversity Intactness Index
+      (BII)
+    unit: '%'
+    weight: Land Cover
+    navigate: Terrestrial Biodiversity|BII
+    tier: 2
+- Terrestrial Biodiversity|Biodiversity Intactness Index|Cropland Landscapes:
+    description: Terrestrial biodiversity in landscapes containing cropland measured
+      with Biodiversity Intactness Index (BII)
+    unit: '%'
+    sdg: 15
+    skip-region-aggregation: true
+    weight: Land Cover|Cropland
```

Comment:
This is copy-pasting a mistake already in the sag.yaml file - specifying a weight but also stating to skip the region-aggregation is a bit of a contradiction (in nomenclature, the **skip-region-aggregation** specifier wins). 

Are you sure you want to skip the region aggregation if you know the weight?

Reply:
Thanks for spotting. I changed to weight to "Land Cover" for all biodiversity variables. Otherwise, I see the risk that results are not comparable.